examples/transfer/plot_rpa_steps.py

```diff
@@ -0,0 +1,205 @@
+"""
+====================================================================
+Plot the data transformations in the Riemannian Procrustes Analysis
+====================================================================
+
+Use the SpectralEmbedding module to plot in 2D the transformations on the data
+points from source and target domains when applying the Riemannian Procrustes
+Analysis to match their statistics.
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+from pyriemann.datasets import sample_gaussian_spd
+from pyriemann.embedding import SpectralEmbedding
+from pyriemann.utils.mean import mean_riemann
+from pyriemann.utils.base import invsqrtm, sqrtm, powm, expm
+
+from sklearn.utils import check_random_state
+
+from pyriemann.transfer import (
+    encode_domains,
+    decode_domains,
+    TLCenter,
+    TLRotate,
+)
+
+
+def make_example_transfer_learning(N, class_sep=3.0, class_disp=1.0,
```

Comment:
this function is really copy pasted (I think) in the two examples. Should have a public function for this in simulated.py ?